dnsapi/dns_freemyip.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env sh
+#
+#Author: Recolic Keghart <root@recolic.net>
+#
+########  Public functions #####################
+
+freemyip_token=90c55c84d72bf53f87e65a3c
+
+#Usage: dns_freemyip_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_freemyip_add() {
+  fulldomain="$1"
+  txtvalue="$2"
+
+  _info "Add TXT record using freemyip.com api"
+  _info "WARNING: If your DDNS domain is 'MYDOMAIN.freemyip.com', you can only create certificate for 'MYDOMAIN.freemyip.com' or '*.MYDOMAIN.freemyip.com'. Subdomain is not supported. "
+  _debug "fulldomain: $fulldomain"
+  _debug "txtvalue: $txtvalue"
+
+  freemyip_token="${freemyip_token:-$(_readaccountconf_mutable freemyip_token)}"
```

Comment:
the variable name should in upper case, like:  FM_Token

Reply:
resolved in latest commit.